packages/wrangler/src/__tests__/helpers/mock-cfetch.ts

```diff
@@ -43,12 +59,14 @@ export async function mockFetchInternal(
 		if (uri !== null && (!method || method === (init.method ?? "GET"))) {
 			// The `resource` regular expression will extract the labelled groups from the URL.
 			// These are passed through to the `handler` call, to allow it to do additional checks or behaviour.
-			return await handler(uri, init, queryParams); // TODO: should we have some kind of fallthrough system? we'll see.
+			return await handler(uri, init, queryParams ?? new URLSearchParams()); // TODO: should we have some kind of fallthrough system? we'll see.
 		}
 	}
-	throw new Error(
-		`no mocks found for ${init.method ?? "any HTTP"} request to ${resource}`
-	);
+	// no mocks found for ${init.method ?? "any HTTP"} request to ${resource}
+	// let it fall through to mock-service-worker
+	// (do a real, unmocked fetch)
```

Comment:
Is this line correct? It is not doing a real unmocked fetch, right? All fetches, one way or another, should be mocked in tests.

Reply:
That is what I mean. On the line above you say "fall through to the mock-service-worker" but on this line you say "do a real unmocked fetch". The fetch is still mocked, right? But by the MSW.
Perhaps this line could read, "(do a real, unmocked cfetch)"?

Reply:
the actual unharmed global `fetch` (or the one from fetch-node) is really used and MSW only extends the `end` method of `class NodeClientRequest extends ClientRequest`, see https://github.com/mswjs/interceptors#readme

Reply:
Ah thanks for the clarification. In that case, I would just remove this line. It doesn't add any value, given that we have already stated we are falling through to the MSW.